### PR TITLE
Add new xorg lib

### PR DESCRIPTION
**Pull request by lovedheart**
tested with fresh crew
using "crew install xorg_lib"

### CODE REVIEW

**Comment by lovedheart** [comment]
#1825

File: packages/xorg_lib.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'package'
```
Comment (uberhacker) [inline packages/xorg_lib.rb]: This should have a description, homepage and version.  Take a look at `buildessential` for an example.  Everything else looks great, however.

Comment (lovedheart) [reply packages/xorg_lib.rb]: Fix it.

File: packages/xorg_lib.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+require 'package'
+
```
Comment (uberhacker) [inline packages/xorg_lib.rb]: Missing `require 'package'`?

File: packages/xorg_lib.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on 'libxft'
+  depends_on 'libxmu'
+  depends_on 'libxpm'
+  depends_on 'libxrandr'
+  depends_on 'libxres'
+  depends_on 'libxt'
+  depends_on 'libxv'
+  depends_on 'libxvmc'
+  depends_on 'libxxf86dga'
+  depends_on 'libxxf86vm'
```
Comment (uberhacker) [inline packages/xorg_lib.rb]: It might help to sort these alphabetically to ensure there are no duplicates.

**Pull request merged**